src/auth/auth.ts

```diff
@@ -225,10 +225,13 @@ export class Auth implements AuthService, ConnectionManager {
 
             const isLinkable = (
                 entry: [string, StoredProfile<Profile>]
-            ): entry is [string, StoredProfile<SsoProfile>] =>
-                entry[1].type === 'sso' &&
-                hasScopes(entry[1], ssoAccountAccessScopes) &&
-                entry[1].metadata.connectionState === 'valid'
+            ): entry is [string, StoredProfile<SsoProfile>] => {
+                const r =
```

Comment:
Can we just return instead of assigning to `const r`?

Reply:
This is intentional for debugging. Hard to debug dense one-liners.